src/tinybox.vue

```diff
@@ -202,6 +202,30 @@ export default {
         window.removeEventListener('keyup', this.keyup);
       }
     },
+    
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/NickKaramoff/vue-tinybox/pullRequest?prid=5742748)